src/connection.rs

```diff
@@ -61,6 +61,16 @@ pub struct HttpConnection<T> {
     payload_max_size: usize,
 }
 
+impl<T> AsRawFd for HttpConnection<T>
+where
+    T: AsRawFd,
+{
+    fn as_raw_fd(&self) -> RawFd {
+        self.stream.as_raw_fd()
+    }
+}
+
+// impl<T: Read + Write + ScmSocket> HttpConnection<T> {
```

Comment:
This line should be removed.

Reply:
yes, all the commented code should be cleaned up